contracts/strategies/ConverterStrategyBaseLib.sol

```diff
@@ -1458,17 +1542,25 @@ library ConverterStrategyBaseLib {
     return IPriceOracle(IConverterController(converter.controller()).priceOracle()).getAssetPrice(token);
   }
 
-  function registerIncome(uint assetBefore, uint assetAfter, uint earned, uint lost) internal pure returns (
-    uint _earned,
-    uint _lost
-  ) {
+  function registerIncome(uint assetBefore, uint assetAfter) internal pure returns (uint earned, uint lost) {
     if (assetAfter > assetBefore) {
-      earned += assetAfter - assetBefore;
+      earned = assetAfter - assetBefore;
     } else {
-      lost += assetBefore - assetAfter;
+      lost = assetBefore - assetAfter;
     }
     return (earned, lost);
   }
+
+  /// @notice Register income and cover possible loss
+  function coverPossibleStrategyLoss(uint assetBefore, uint assetAfter, address splitter) external returns (uint earned) {
+    uint lost;
+    (earned, lost) = ConverterStrategyBaseLib.registerIncome(assetBefore, assetAfter);
+    if (lost != 0) {
+      ISplitter(splitter).coverPossibleStrategyLoss(earned, lost);
```

Comment:
I think need to limit the amount similar to sending excess assets
`amountToSend = Math.min(amountToSend, PRICE_CHANGE_PROFIT_TOLERANCE * strategyBalance / 100_000);`
coz otherwise in a case of price spike it will lead to weird behaviour

Reply:
so in `FixChangePrices` will need to add exactly amount